libs/output-mapping/src/Storage/TableStructureValidator.php

```diff
@@ -132,6 +145,41 @@ private function validateTypedTable(array $table, array $schemaColumns, string $
         }
     }
 
+    /**
+     * @param MappingFromConfigurationSchemaColumn[] $schemaColumns
+     */
+    private function validateUntypedTable(array $table, array $schemaColumns): void
+    {
+        $validationErrors = [];
+        foreach ($schemaColumns as $schemaColumn) {
+            if (!$schemaColumn->getDataType()) {
+                continue;
+            }
+
+            try {
+                $schemaColumn->getDataType()->getTypeName($table['bucket']['backend']);
```

Comment:
otázkou je jestli to řešit až takhle no
1/ pokud je u netypovaný tabulky zadaný typ pro konkrétní backend tak to spadn
2/ nebo pokud je basetype jiný než STRING tak to spadne

Reply:
To je popsané tady v tom dokumentu (poslední bod)
https://keboola.atlassian.net/wiki/spaces/ENGG/pages/3099787265/PENDING+RFC-2023-016+Native+Data+types+Take+2#Output-processing-Logic

> except if the component is producing a typed table and the destination table is not typed - and the table structure is the same - which means there is same number of same named columns - then the job succeeds (with warning?) 

> This is to maintain backward compatibility with existing projects.

Reply:
no to je, ale na tom callu jsme se domluvili že by se typy mohly kontrolovat oproti base_type

https://drive.google.com/file/d/1S5r-QqtX9x9BwCrHMfCk8P1ikHrSJCzP/view?pli=1
někde okolo 57minuty 

proto jsem to udělal takto

Reply:
Si to budu muset fakt naposlouchat :D Už si to nepamatuju.

Reply:
Uprav to prosím jak jsme se dohodli na callu, aby to místo vyjímky zalogovalo warning